cli/src/commands/deploy.ts

```diff
@@ -115,8 +120,15 @@ export default class Deploy extends Command {
     return app.id;
   }
 
+  public log(str: string) {
+    const { flags: { json } } = this.parse(Deploy);
+    if (!json) {
+      super.log(str);
```

Comment:
`log` method is defined as [log(message?: string, ...args: any[])](https://github.com/oclif/command/blob/6be030b420f17c861823ba68f3f0d74cedd3852a/src/command.ts#L132) - we never used the extra arguments, but probably best to keep supporting them.

Reply:
Will do, was basing the method signature according to the [Logger interface](https://github.com/reshufflehq/reshuffle/blob/96ab87bb4ff2c94b57ee79d35c9210812ba1a8a1/build-utils/src/index.ts#L12) but using the interface for oclif's log is the more correct way